client/src/components/Indicators.js

```diff
@@ -392,6 +392,9 @@ class IndicatorsWithoutI18n extends Component {
 
                 <div className="Indicators__viz">
                   <button
+                    aria-hidden={
+                      this.state.xAxisStart === 0 || this.state.activeTimeSpan === "year"
+                    }
```

Comment:
Oh neat, I didn't realize you could actually just pass a boolean to this prop. In the tenants platform I have a helper function that converts a boolean to the strings `"true"` and `"false"`, I think.

Reply:
Yeah! Same with tabIndex too I think :)